packages/compass-e2e-tests/tests/collection-import.test.ts

```diff
@@ -26,16 +27,20 @@ async function importJSONFile(browser: CompassBrowser, jsonPath: string) {
   const importModal = await browser.$(Selectors.ImportModal);
   await importModal.waitForDisplayed();
 
-  // Confirm
+  // Confirm import.
   await browser.clickVisible(Selectors.ImportConfirm);
 
-  // wait for the done button to appear and then click it
-  const doneButton = await browser.$(Selectors.ImportDone);
-  await doneButton.waitForDisplayed();
-  await browser.clickVisible(Selectors.ImportDone);
+  // Wait for the modal to go away.
+  await importModal.waitForDisplayed({ reverse: true });
 
-  // wait for the modal to go away
-  await importModal.waitForDisplayed({ reverse: false });
+  // Wait for the done toast to appear and close it.
+  const toast = Selectors.ImportSucceededToast;
+  await browser.$(toast).waitForDisplayed();
+  await browser.$(Selectors.closeToastButton(toast)).waitForDisplayed();
+  await browser.waitForAnimations(toast);
+  await browser.clickVisible(Selectors.closeToastButton(toast));
```

Comment:
`clickVisible` does `waitForAnimations` internally, I am a bit surprised that you need to wait first and then click 🤔

Reply:
Ah yup. This was not needed.